### ISSUE TITLE
teal but with home button exit

### ISSUE THREAD

**Issue by SalTeyJek004** [libertyernie/Nintendont#21]
i think its weird that home button exit is turned off
call it like, blue or cerulean

**Comment by libertyernie** [comment]
Good idea (if I can get it to work - I remember having problems with making the home button exit work when building using newer libraries).

Right now the home button exit is disabled on all the newest builds, by the way.

**Comment by SalTeyJek004** [comment]
yeah i switched to green hoping it was just a teal thing
but alas

**Comment by terminusthree** [comment]
Not sure if you plan on revisiting this project but I would also love this feature , more so on black or gray builds. Thanks for these custom builds and Nicoe.

**Comment by libertyernie** [comment]
I was just thinking the other day that it might be nice to make new builds based off a newer Nintendont version. I might not make as many builds as before but if I do make some, I'll take a look at the GitHub issues like this one and see if there's anything I ought to clean up, and putting home button exit back in (by turning off the LI_NOEXIT compiler flag) will probably be one of the things I do.

**Comment by terminusthree** [comment]
Not sure if you've seen it, but I saw a new build by [SuperrSonic ](https://youtu.be/BbS-w6YMeiI) recently that may be worth looking at, but it's a separate fork.

**Comment by libertyernie** [comment]
Thanks. Maybe someday I'll make a build based off that. It looks like it can already launch specific GameCube games directly from the HBC, and I think it changes the exit button to the power button on the console. The only other things I'd want are the custom Classic Controller mappings, although at this point I'd probably use a preprocessor flag for each mapping I need, and probably only implement it for the games I have on my console - it's been a while since I've worked on this and I don't want to deal with memory limits again if my own use case doesn't force it.
